packages/renderer/src/lib/kube/pods/terminal/KubernetesTerminalService.spec.ts

```
@@ -18,23 +18,23 @@
-import type { PodContainerInfo, PodInfo } from '@podman-desktop/api';
+import type { PodInfoContainerUI, PodInfoUI } from '@podman-desktop/core-api';
 import { beforeEach, expect, test } from 'vitest';
 
 import { TerminalService } from '/@/lib/kube/pods/terminal/KubernetesTerminalService';
 
 let terminalService: TestableKubernetesTerminalService;
 
 class TestableKubernetesTerminalService extends TerminalService {
-  public testInvalidateCacheRecordOnStatusUpdate(podsInfos: PodInfo[]): void {
+  public testInvalidateCacheRecordOnStatusUpdate(podsInfos: PodInfoUI[]): void {
     return this.invalidateCacheRecordOnStatusUpdate(podsInfos);
```

Biome (2.5.1)

[error] The function should not return a value because its return type is void.

(lint/correctness/noVoidTypeReturn)